components/Notes/index.js

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+
+export default function Notes({ recipeId }) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [note, setNote] = useState("");
+
+  useEffect(() => {
+    const savedNote = localStorage.getItem(`note-${recipeId}`);
+    if (savedNote) {
+      setNote(savedNote);
+      setIsOpen(true);
+    }
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem(`note-${recipeId}`, note);
+  }, [note]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+  };
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const handleChange = (event) => {
+    setNote(event.target.value);
+  };
+
+  return (
+    <>
+      <StyledButton type="button" onClick={handleToggle}>
+        {isOpen ? "Close Note" : "Add Note"}
+      </StyledButton>
+
+      {isOpen && (
+        <StyledForm onSubmit={handleSubmit}>
+          <StyledTextarea
+            value={note}
+            name="notes"
+            onChange={handleChange}
+            maxLength={500}
+          />
+          <br />
```

Comment:
Again: please don't use `<br>` elements to create distance, this is not the semantics of this element.